css-view-transitions-1/Overview.bs

```diff
@@ -1275,17 +1284,8 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 				or |element| is [=element-not-rendered|not rendered=],
 				then [=continue=].
 
-			1. If any of the following is true:
-
-				* |usedTransitionNames| [=list/contains=] |transitionName|.
-
-				* |element| is not |element|'s [=tree/root=]
-					and |element| does not have [=layout containment=].
-
-				* |element| is not |element|'s [=tree/root=]
-					and |element| allows [=fragmentation=].
-
-				Then return failure.
```

Comment:
Moved to "Update pseudo-element styles", since these checks need to happen per frame.

Reply:
Sounds good.